components/common/AccountSelection.js

```diff
@@ -26,6 +28,12 @@ const AccountSelection = ({
 		addToLocalStorage(networkInfo.network, "selectedAccount", account.address);
 		track(goalCodes.GLOBAL.ACCOUNT_SELECTED);
 		setIsStashPopoverOpen(false);
+
+		axios
+			.get(`/${networkInfo.network}/user/existing-user/${account.address}`)
+			.then(({ data }) => {
+				setIsExistingUser(data.isExistingUser);
+			});
```

Comment:
@Lord-of-Codes add catch statement log the error using `console.error`

Reply:
made the chanes